public/employers/csec.yml

```diff
@@ -0,0 +1,36 @@
+name: Calgary Sports and Entertainment Corporation
+aliases:
+  - Calgary Flames
+shortName: CSEC
+wiki: Calgary Sports and Entertainment
+image: csec.svg
+officialWebsite: http://news.gamestop.com/about-gamestop
```

Comment:
Wrong official site I think

Reply:
Turns out they don't have an official website and just do everything through NHL, so removing this.